### PR TITLE
Feature/setup

### PR DESCRIPTION
**Pull request by Bawanthathilan**
image

### CODE REVIEW

File: src/modules/HomeModules/eventCard/eventCard.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        />
+        <div className="card-body">
+          <h5 className="card-title">Event Name</h5>
+          <p className="card-text">Description</p>
+        </div>
+      </div>
+    </>
+  );
+}
+
```
Comment (chamodshehanka) [inline src/modules/HomeModules/eventCard/eventCard.tsx]: Change eventCard/eventCard.tsx to EventCard/EventCard.tsx 
capital letters @Bawanthathilan

File: src/modules/HomeModules/upComingEvents/upComingEvents.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          </h2>
+
+          <div className="col-12">
+            <div id="carousel" className="carousel slide" data-ride="carousel">
+              <ol className="carousel-indicators">
+                <li
+                  data-target="#carousel"
+                  data-slide-to={0}
+                  className="active bg-secondary"
+                />
```
Comment (chamodshehanka) [inline src/modules/HomeModules/upComingEvents/upComingEvents.tsx]: Component, FileName and Folder should start with caps

**Review by chamodshehanka — CHANGES_REQUESTED** [review]

**Comment by chamodshehanka** [comment]
@Bawanthathilan ko okkoma capital dala nane component wala and components related scss files ehema

**Comment by Bawanthathilan** [comment]
> 
> 
> @Bawanthathilan ko okkoma capital dala nane component wala and components related scss files ehema

Dn balapn 🤯

**Review by chamodshehanka — APPROVED** [review]

**Pull request merged**